Tribes/src/app/building.ts

```diff
@@ -1,14 +1,14 @@
 export class Building {
     id: number;
-    type: string;
+    buldingTypeENUM: string;
```

Comment:
It is not just `type`?

Reply:
The json, which we get from the backend, has the field 'buildingTypeENUM'.